### DISCUSSION TITLE
Using static mode, when navigation routes, it's empty

### DISCUSSION THREAD

**Discussion by CHEWX** [nuxt/nuxt#23745] [Questions (Nuxt 2)]
### Environment

`nuxt: ^2.15.8 => 2.15.8`

Hosted on Netlify.

### Reproduction

I cannot replicate, but this is what someone sent me.

https://www.veed.io/view/b6787da1-ab42-40a0-a523-d5a3baf24d1f?panel=share

Website: https://odorne.com

### Describe the bug

Website using `static` mode and deployed using `npm run generate`

**Comment by danielroe** [comment]
Would you provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Comment by CHEWX** [comment]
I was hoping the video was sufficient enough to show what's going on and also the live website?

I guess just an opinion on what is happening in the video would suffice?
